### DISCUSSION TITLE
Infinite "processing information" in uploading proof of getting student developer pack

### DISCUSSION THREAD

**Discussion by GazimSoliev** [community/community#86102] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/63443274/06a4165f-aa15-46c9-827d-609b7279bfda)

**Comment by soniasingh12** [comment]
For me was the image that I uploaded too large, I used Image resizer from PowerToys and then it worked

**Comment by mishmanners** [comment, marked as answer]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:
- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file less than 150kB
